include/config/backend/command_line_arguments.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <map>
+#include <ranges>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <config/config_backend.h>
+#include <config/config_specification.h>
+
+namespace silo::config {
+
+class CommandLineArguments : public ConfigBackend {
+   std::vector<std::string> args;
+
+  public:
+   explicit CommandLineArguments(std::span<const std::string> args_)
+       : args(args_.begin(), args_.end()) {}
+
+   [[nodiscard]] constexpr std::string_view errorContext() const {
+      return "command line arguments";
+   };
```

Comment:
This is used in a single place, should we inline it?

Reply:
It's implementing an interface now (again).